images/win/scripts/ImageHelpers/VisualStudioHelpers.ps1

```diff
@@ -72,12 +72,18 @@ function Get-VsCatalogJsonPath {
     return Join-Path $instanceFolder.FullName "catalog.json"
 }
 
+function Get-VisualStudioInstance {
+    # Use -Prerelease and -All flags to make sure that Preview versions of VS are found correctly
+    $vsInstance = Get-VSSetupInstance -Prerelease -All | Where-Object { $_.DisplayName -match "Visual Studio" } | Select-Object -First 1
+    $vsInstance | Select-VSSetupInstance -Product *
+}
+
 function Get-VisualStudioPath {
-    return (Get-VSSetupInstance | Select-VSSetupInstance -Product *).InstallationPath
+    return (Get-VisualStudioInstance).InstallationPath
 }
 
 function Get-VisualStudioPackages {
-    return (Get-VSSetupInstance | Select-VSSetupInstance -Product *).Packages
+    return (Get-VisualStudioInstance).Packages
```

Comment:
Do we really need these 1-line functions? Maybe use (Get-VisualStudioInstance).Packages and (Get-VisualStudioInstance).InstallationPath in the scripts as we do in the software report?